ci/prow/config_start.yaml

```diff
@@ -236,26 +236,66 @@ metadata:
 kind: Role
 apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
+  namespace: default
   name: "sinker"
 rules:
   - apiGroups:
-      - ""
+      - "prow.k8s.io"
     resources:
-      - pods
+      - prowjobs
     verbs:
       - delete
       - list
+      - watch
+      - get
   - apiGroups:
-      - "prow.k8s.io"
+      - ""
     resources:
-      - prowjobs
+      - configmaps
+    resourceNames:
+      - prow-sinker-leaderlock
+    verbs:
+      - get
+      - update
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+      - events
+    verbs:
+      - create
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  namespace: test-pods
+  name: "sinker"
+rules:
+  - apiGroups:
+      - ""
```

Comment:
Does it make sense to add a comment "" indicates the core API group. I find it not-intuitive and only figure out what it does after i read the doc. (could just be because i'm not a k8s expert :( 
https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole

Reply:
Ditto.